core/src/components.d.ts

```diff
@@ -1391,10 +1392,15 @@ export namespace Components {
           * Animation to use when the loading indicator is presented.
          */
         "enterAnimation"?: AnimationBuilder;
+        "hasController": boolean;
         /**
           * Additional attributes to pass to the loader.
          */
         "htmlAttributes"?: LoadingAttributes;
+        /**
+          * If `true`, the loading indicator will open. If `false`, the loading indicator will close. Use this if you need finer grained control over presentation, otherwise just use the loadingController or the `trigger` property. Note: `isOpen` will not automatically be set back to `false` when the loading indicator dismisses. You will need to do that in your code.
```

Comment:
`trigger` is not supported as of this pull request, but I opted it for aligning the documentation with `ion-modal` now vs. risking forgetting to update this later. When `FW-2334` lands in `feature-7.0`, this will be accurate.